rpc_jobs/rpc_gating_lint.yml

```diff
@@ -18,6 +19,11 @@
         deleteDir()
         stage("Prepare"){
           git url: env.ghprbAuthorRepoGitUrl, branch: ghprbSourceBranch
+          /* Set mtime to a constant value as git doesn't track mtimes but
```

Comment:
I think it would be nice to create a function for this so we're not copying the same thing around.